RFIDKit/ISO - Fix SIZE comparison warning

```
[253/450] Building CXX object libs/ActivityKit/CMakeFiles/ActivityKit.dir/source/activities/ChooseReinforcer.cpp.obj
In file included from /Users/ladislas/dev/leka/LekaOS/libs/RFIDKit/include/RFIDKit.h:7,
                 from /Users/ladislas/dev/leka/LekaOS/libs/ActivityKit/include/activities/ChooseReinforcer.h:9,
                 from /Users/ladislas/dev/leka/LekaOS/libs/ActivityKit/source/activities/ChooseReinforcer.cpp:7:
/Users/ladislas/dev/leka/LekaOS/libs/RFIDKit/include/ISO14443A.h: In instantiation of 'std::array<unsigned char, (SIZE + 1)> leka::rfid::Command<SIZE>::getArray() const [with unsigned int SIZE = 1]':
/Users/ladislas/dev/leka/LekaOS/libs/RFIDKit/include/ISO14443A.h:145:106:   required from here
/Users/ladislas/dev/leka/LekaOS/libs/RFIDKit/include/ISO14443A.h:23:21: warning: comparison of integer expressions of different signedness: 'int' and 'unsigned int' [-Wsign-compare]
   23 |   for (int i = 0; i < SIZE; ++i) {
      |                   ~~^~~~~~
/Users/ladislas/dev/leka/LekaOS/libs/RFIDKit/include/ISO14443A.h: In instantiation of 'std::array<unsigned char, (SIZE + 1)> leka::rfid::Command<SIZE>::getArray() const [with unsigned int SIZE = 2]':
/Users/ladislas/dev/leka/LekaOS/libs/RFIDKit/include/ISO14443A.h:151:65:   required from here
/Users/ladislas/dev/leka/LekaOS/libs/RFIDKit/include/ISO14443A.h:23:21: warning: comparison of integer expressions of different signedness: 'int' and 'unsigned int' [-Wsign-compare]
```

--- a/libs/RFIDKit/include/ISO14443A.h
+++ b/libs/RFIDKit/include/ISO14443A.h
@@ -11,7 +11,7 @@
 
 namespace leka::rfid {
 
-template <size_t SIZE>
+template <std::size_t SIZE>
 struct Command {
 	const std::array<uint8_t, SIZE> data;
 	const leka::rfid::Flag flags;
@@ -20,7 +20,7 @@ struct Command {
 	{
 		auto cmd = std::array<uint8_t, SIZE + 1> {};
 
-		for (int i = 0; i < SIZE; ++i) {
+		for (std::size_t i = 0; i < SIZE; ++i) {
 			cmd.at(i) = data.at(i);
 		}
 

